app/ui/legacy/src/main/java/com/fsck/k9/ui/messagelist/MessageListSwipeCallback.kt

```diff
@@ -171,7 +171,8 @@ class MessageListSwipeCallback(
     }
 
     private fun Canvas.drawBackground(dX: Float, width: Int, height: Int) {
-        val swipeActionConfig = if (dX > 0) swipeRightConfig else swipeLeftConfig
+        val swipeActionConfig = (if (dX > 0) swipeRightConfig else swipeLeftConfig)
+            ?: error("drawBackground() called despite swipeActionConfig == null")
```

Comment:
These lines could be extracted twice:

``` kotlin
private fun swipeActionConfigOf(swipeRight: Boolean): SwipeActionConfig {
    val config = if (swipeRight) swipeRightConfig else swipeLeftConfig
    return config ?: error("drawBackground() called despite swipeActionConfig == null")
}
```

Also, I find it more readable in two lines.

Reply:
Moved the error checking to a separate line.